.github/workflows/tests.yml

```
@@ -0,0 +1,29 @@
+name: Python package
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+
+      - name: Display Python version
+        run: python -c "import sys; print(sys.version)"
+
+      - name: Install dependancies
+        run: |
+          pip install pipenv
+          pipenv install --dev
```

I would keep  the `--deploy` option to validate and use the `Pipfile.lock` file to install dependencies
```suggestion
          pipenv install --dev --deploy
```

---

That's probably the only think that I could have forgotten ... --'

---

you force-pushed, but no change is visible :slightly_smiling_face:

---

Well, that's interesting ^^'